Source/Core/Core/HW/DSPHLE/UCodes/Zelda.cpp

```diff
@@ -201,36 +201,38 @@ void ZeldaUCode::HandleMailDefault(u32 mail)
   case MailState::WAITING:
     if (mail & 0x80000000)
     {
-      if ((mail >> 16) != 0xCDD1)
+      if ((mail & TASK_MAIL_MASK) != TASK_MAIL_TO_DSP)
       {
         PanicAlertFmt("Rendering end mail without prefix CDD1: {:08x}", mail);
```

Comment:
Should this still be considered a Panic Alert?

Reply:
It's still something that shouldn't ever happen unless things get desynced (in which case it'd break on LLE too probably), but it's probably fine to convert to a warning log instead. I'll do that.

Reply:
Changed in #10753.